src/Umbraco.Core/Services/EntityXmlSerializer.cs

```diff
@@ -322,8 +338,9 @@ public XElement Serialize(IMacro macro)
         /// </summary>
         /// <param name="dataTypeService"></param>
         /// <param name="contentType">Content type to export</param>
+        /// <param name="folders">The path of folders for this content type separated by a backslash, for example: `SEO/Meta`</param>
         /// <returns><see cref="XElement"/> containing the xml representation of the IContentType object</returns>
-        public XElement Serialize(IDataTypeService dataTypeService, IContentType contentType)
+        public XElement Serialize(IDataTypeService dataTypeService, IContentType contentType, string folders = "")
```

Comment:
we unfortunately cannot simply add optional parameters to a method since that will cause YSODs when a project hasn't been rebuilt against this codebase, the only option is to add an overload (i'll update)